### DISCUSSION TITLE
Buttons for group images

### DISCUSSION THREAD

**Discussion by Marcelluxx** [python-telegram-bot/python-telegram-bot#3922] [Q&A]
Hi everyone, I have the following question:
Can I send a group of 2 images with a caption for the first image and adding below two buttons?

**Comment by Poolitzer** [comment]
The caption will appear below the group of images, and then below that will be the buttons

**Reply by Marcelluxx** [reply]
Can you give me an example with the code?

**Reply by clot27** [reply]
you can use the [send_media_group](https://docs.python-telegram-bot.org/telegram.bot.html#telegram.Bot.send_media_group) method.

**Reply by Bibo-Joshi** [reply]
@Poolitzer keyboards can not be attached to `send_media_group` ;)

**Reply by Poolitzer** [reply]
Oh wow I thought I have seen that. Sorry.

**Reply by Marcelluxx** [reply]
> @Poolitzer keyboards can not be attached to `send_media_group` ;)

So there is no methods, right ?

**Reply by Poolitzer** [reply]
Apparently not

**Reply by Marcelluxx** [reply]
😕
